### PR TITLE
Fix search when explicitly passed region=None

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Comment by robhudson** [comment]
This fixes the issue found here:
http://sentry.mktmon.services.phx1.mozilla.com/mkt/marketplacefirefoxcom/group/16335/

**Comment by chuckharmston** [comment]
Is there a bug number for this? Commit message doesn't really describe what was broken.

**Comment by robhudson** [comment]
No, just a fix for a sentry error I saw. I could file one and attach it tho which is probably a better idea.

File: mkt/search/tests/test_views.py [hunk 1]
```diff
@@ -1389,6 +1389,12 @@ def test_search_preferred_region_match(self):
         objs = res.json['objects']
         eq_(len(objs), 0)
 
+    def test_region_explicit_none(self):
+        res = self.anon.get(self.url, data={'region': 'None'})
```
Comment (chuckharmston) [inline mkt/search/tests/test_views.py]: It's really weird that the querystring parameter `None` is turned into the Python `None` value. Where does that happen?

Comment (robhudson) [inline mkt/search/tests/test_views.py]: https://github.com/mozilla/zamboni/blob/be1aa96d5e24fe398c18929732f4cac4f75769a8/mkt/api/base.py#L51-L52

Comment (chuckharmston) [inline mkt/search/tests/test_views.py]: I guess that works. I'd have used 0, but ¯\_(ツ)_/¯

**Comment by chuckharmston** [comment]
r+ if tests pass, looks good.

**Pull request merged**